pagesections/main.statement.section.js

```diff
@@ -210,10 +209,10 @@ const MainStatementSection = ( Base ) => class extends Base {
 	 * @private
 	 * @param {int} index
 	 * @param {string} propertyId
-	 * @return {element}
+	 * @return {Promise<element>}
```

Comment:
Is this the correct way to typehint this sort of async/await method?

Reply:
In theory yes, although I don't know where `element` as a type is coming from and it doesn't seem to help my IDE very much. `Promise<WebdriverIO.Element>` seems to work.